lib/config/options/index.ts

```diff
@@ -120,6 +120,36 @@ const options: RenovateOptions[] = [
       'Change this value in order to override the default onboarding PR title.',
     type: 'string',
     default: 'Configure Renovate',
+    cli: false,
+  },
+  {
+    name: 'configMigration',
+    description: 'Enable this to get Config migration PRs when needed.',
+    type: 'boolean',
+    default: false,
+  },
+  {
+    name: 'configMigrationBranch',
```

Comment:
do we really need all those new config options? can't we override existing for options for migration?

Reply:
removed configMigration[PrTitle | CommitMessage], now using existing commitMessage instead

Reply:
No, probably not